app/components/container/form-volumes/component.js

```diff
@@ -12,6 +12,7 @@ export const EXISTING_PVC = 'existingPvc';
 export const BIND_MOUNT = 'bindMount';
 export const TMPFS = 'tmpfs';
 export const SECRET = 'secret';
+export const CERTIFICATE = 'certificate';
```

Comment:
We're adding more and more to this list, can we move this to the constants file? You'll need to find the spots we use these and change those as well.